contracts/FlowIDTableStaking.cdc

```diff
@@ -716,7 +716,7 @@ pub contract FlowIDTableStaking {
 
                 let nodeRecord = FlowIDTableStaking.borrowNodeRecord(nodeID)
 
-                if nodeRecord.tokensStaked.balance == 0.0 { continue }
+                if nodeRecord.tokensStaked.balance == 0.0 || nodeRecord.role == UInt8(5) { continue }
```

Comment:
this is to say access node will not get reward even if they staked?

Reply:
Yes, I had forgotten to not allow them to get rewards. Thank you for pointing that out earlier!